Formula/masscan.rb

```diff
@@ -31,6 +31,6 @@ def install
   end
 
   test do
-    assert_match "ports =", shell_output("#{bin}/masscan --echo | head -n 6 | tail -n 1")
+    assert_match "ports =", shell_output("#{bin}/masscan --echo | head -n 8 | tail -n 1")
```

Comment:
Since this is an `assert_match`, why do we need the calls to `head` and `tail`?

Reply:
Good point, we don't, and removing them will make this less prone to breakage. 😍